example/src/Onboarding.tsx

```diff
@@ -41,7 +51,13 @@ export const OnboardingEOR = () => {
   return (
     <RemoteFlows auth={fetchToken}>
       <div className="cost-calculator__container">
-        <OnboardingFlow employmentId={'1234'} render={MultiStepForm} />
+        <OnboardingFlow
+          employmentId="1234"
+          countryCode="PRT"
+          employeeType="employee"
```

Comment:
I think we should rename this to `type`, ie same name as we have in `EmploymentCreateParams`. We just need to document the props in `OnboardingFlowProps` component.